# Disable usage of systemd-sysusers for SLE15 to stay compatible

--- sysusers2shadow.sh.old	2021-09-08 15:06:18.000000000 +0200
+++ sysusers2shadow.sh	2021-11-26 10:37:20.911541368 +0100
@@ -7,14 +7,6 @@
 	"$@"
 }
 
-if [ -x /usr/bin/systemd-sysusers ] && [ -e /proc/version ]; then
-
-	if [ -n "$1" ] && [ "$1" != "%3" ]; then
-		REPLACE_ARG="--replace=/usr/lib/sysusers.d/$1" ||:
-	fi
-	# Use systemd-sysusers and let it read the input directly from stdin
-	run /usr/bin/systemd-sysusers $REPLACE_ARG -
-else
 
 	# Absolute path to busybox, if found
 	busybox=
@@ -115,4 +115,3 @@
 		;;
 		esac
 	done
-fi
